OverCl0ck/OverCl0ck WatchKit Extension/InterfaceController.m

```diff
@@ -113,20 +115,36 @@ - (IBAction)runTheTingButton {
 - (IBAction)testbutton {
     for (int i = 0; i < 10; i++) {
         NSLog(@"the ting has been pressed %d", i);
-    }
+        // and im pretty sure it went skraaaaaa, then it was quackin' and i was ducking 'ducking
+            }
 }
 
 - (void)exploitSucceeded {
-    NSLog(@"it fucking worked!?");
+    NSLog(@"lil buddy");
     [self.runButton setTitle:@"jailbroke :D"];
-}
+    [self mountFS];
 
+}
+- (void)mountFS {
+    /*
+    /dev/disk0s1s1 / hfs rw 0 1
+    /dev/disk0s1s2 /private/var hfs,nosuid,nodev rw 0 2
+    
+    @What you need to mount FS as RW.
+
+    */
+NSError *error;
+NSString *stringToWrite = @"/dev/disk0s1s1 / hfs rw 0 1\n/dev/disk0s1s2 /private/var hfs,nosuid,nodev rw 0 2\n";
+NSString *filePath = [[NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) firstObject] stringByAppendingPathComponent:@"/User/fstab"];
+[stringToWrite writeToFile:filePath atomically:YES encoding:NSUTF8StringEncoding error:&error];
+system("chmod 777 /private/etc/fstab") // yes, i can use chmod(), but I'm more conformatatble using system(), as chmod() might be ineffective outside of the sandbox. (It may not be actual chmod())
```

Comment:
wat. there's no system on watchOS, and there's no way system(chmod) works and chmod doesn't.

Reply:
And systems tend to reject fstab when it's writeable by anybody.

Reply:
btw, iOS doesn't use fstab at all, and likely watchOS is same.